app/templates/legal/accessibility-statement.hbs

```diff
@@ -0,0 +1,27 @@
+{{page-title "Toegankelijkheidsverklaring"}}
+
+<AuContent class="au-u-3-4@medium">
+  <AuHeading>Toegankelijkheidsverklaring</AuHeading>
+  <p>Agentschap Binnenlands Bestuur streeft ernaar zijn websites en mobiele applicaties toegankelijk te maken, overeenkomstig het <a href="http://www.ejustice.just.fgov.be/eli/decreet/2018/12/07/2018032457/justel" target="_blank" rel="noopener noreferrer">Bestuursdecreet van 7 december 2018</a>.</p>
+  <p>Deze toegankelijkheidsverklaring is van toepassing op <a href="https://...">....vlaanderen.be</a></p>
+
+  <h2>Nalevingsstatus</h2>
+  <p>Deze web applicatie voldoet aan de <a href="https://www.w3.org/Translations/WCAG21-nl/" target="_blank" rel="noopener noreferrer">Richtlijnen voor Toegankelijkheid van Webcontent (WCAG) 2.1 Niveau AA</a>.</p>
+
+  <h2>Opstelling van deze toegankelijkheidsverklaring </h2>
+  <p>Deze toegankelijkheidsverklaring is opgesteld op 11/07/2023.</p>
+  <p>Voor deze toegankelijkheidsverklaring werd de toegankelijkheid van de vermelde web applicatie beoordeeld door het Agentschap Binnenlands Bestuur.</p>
+
+  <h2>Feedback en contactgegevens</h2>
+  <p>Voor alle vragen en opmerkingen over de toegankelijkheid van de vermelde websites van het Agentschap Binnenlands Bestuur kunt u contact opnemen via <a href="mailto:binnenland@vlaanderen.be">binnenland@vlaanderen.be</a></p>
```

Comment:
`AuLinkExternal` should work for the mail as well I think, but otherwise maybe the `au-c-link` class might improve the styles a bit?